src/components/themeProvider/__tests__/index.test.tsx

```diff
@@ -21,17 +21,21 @@ const renderWrapper = (theme: Props['theme']) =>
 describe('ThemeProvider', () => {
   it('provides as24 theme', () => {
     renderWrapper('as24');
+    const {
+      colors: { brand },
+    } = autoScout24Theme;
+    const castedBrand = brand as Partial<ColorHues>;
```

Comment:
Not fully sure on this one, suggestions are welcome 🙂

Reply:
Why do we need the typecast here?